ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-release-2.5.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: builder
     namespace: stolostron
-    tag: go1.16-linux
+    tag: go1.17-linux
```

Comment:
The `Dockerfile.e2etest` instance below needs to be updated also.

Reply:
Yeah, the `main` branch has been updated. `release-2.5` will be updated after being fast forwarded

Reply:
done